Kenya/KEN_transform_load_dlt.py

```diff
@@ -202,6 +202,16 @@ def boost_silver():
             .when(((~col('Class').startswith('2')) & (col('Chapter_econ2').startswith('24'))), 'Interest on debt')
             # other expenses
             .otherwise('Other expenses')
+        ).withColumn('geo_0.5', 
+            when(col('adm1_name').isin('Kiambu', 'Kirinyaga', 'Murang’A', 'Nyandarua', 'Nyeri', 'Tharaka Nithi'), 'Central')
```

Comment:
@nihaas21 recall we talked about `adm1_name` being a deprecated variable and should not be used in the pipelines; `geo1` should be used instead. 

I see in the case of Kenya here, there's both `geo1` and `adm1_name` defined above. Can you refactor the code to remove `adm1_name` and update all references to `adm1_name` to `geo1`? Note the definition for `adm1_name` is different from `geo1`, check with @bhupatiraju to see if the definition for `adm1_name` needs to be incorporated into `geo1`.